visualdl/logic/sdk.cc

```diff
@@ -352,6 +347,111 @@ std::string TextReader::caption() const {
 
 size_t TextReader::size() const { return reader_.total_records(); }
 
+void Audio::StartSampling() {
+  if (!ToSampleThisStep()) return;
+
+  step_ = writer_.AddRecord();
+  step_.SetId(step_id_);
+
+  time_t time = std::time(nullptr);
+  step_.SetTimeStamp(time);
+
+  // resize record
+  for (int i = 0; i < num_samples_; i++) {
+    step_.AddData();
+  }
+  num_records_ = 0;
+}
+
+int Audio::IsSampleTaken() {
```

Comment:
Minor stuff, the function name is implying that the function will return a BOOL, but the function returns an index. 
Maybe rename the function to `NextRandSampleIndex` or provide a comment here to explain the logic.

Reply:
true